publisher.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk-sqs'
+class Publisher
+	# Note: You can store the values of the aws access keys in a config file (json) 
+	# from where you can get the values into the constants.
+	ACCESS_KEY_ID = #<your access key id>
+	SECRET_ACCESS_KEY = #<your secret access key>
+
+	attr_accessor :sqs 
+	def initialize
+		creds = Aws::Credentials.new(ACCESS_KEY_ID, SECRET_ACCESS_KEY)
+		@sqs = Aws::SQS::Client.new(region: 'ap-south-1', credentials: creds)
+	end
+
+	# For Standard Queue
+	def create_std_q queue_name
+		std_queue = sqs.create_queue(queue_name: queue_name)
+	end
+
+	# For FIFO Queue
+	def create_fifo_q queue_name
+		queue_name = queue_name.split('.')[0]
```

Comment:
Not understood the reason for this line.

Reply:
@gnoronha-mdsol Going to make this configurable too. So the user can specify whether to create a standard or FIFO queue.

Reply:
Out of scope for this PR.